### DISCUSSION TITLE
Native Windows on ARM build

### DISCUSSION THREAD

**Discussion by GewoonJaap** [f1multiviewer/issue-tracker#348] [Feature requests] [answered]
Hi,

I was wondering if you could add a native Windows on ARM build for this application? If you want, I can test it for you. I have a Windows 2023 Dev kit at home, which runs on Windows on ARM.
Which isn't that much different than 64bit Windows 11, this simply has ARM support :).

**Comment by LapsTimeOFF** [comment]
Hey! To be confirmed, Widevine is not available on Windows ARM, this make impossible to watch DRM-protected videos

**Comment by LapsTimeOFF** [comment, marked as answer]
Hey, sorry for the very late notice, Windows ARM builds have been available for a few versions now, you can find them on [our downloads page](https://muvi.gg/dl)
